src/models/stores/serial.ts

```diff
@@ -107,4 +116,28 @@ export class SerialDevice {
       }
     } while (match);
   }
+
+  public writeToOut(n:number){
+    // number visible to user represents "percent closed"
+    // so we need to map x percent to an angle in range where
+    // 100% closed is 120deg, and 0% open is 180deg
```

Comment:
```suggestion
    // 100% (closed) is 120deg, and 0% (open) is 180deg
```
I parsed it incorrectly the first time through.